tests/pr_test.go

```diff
@@ -61,21 +61,27 @@ func TestRunUpgradeExample(t *testing.T) {
 	}
 }
 
-func TestRunFSCloudExample(t *testing.T) {
+func TestRunFSCloudDAExample(t *testing.T) {
 	t.Parallel()
 
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:       t,
-		TerraformDir:  "examples/fscloud",
-		Prefix:        "fscloud",
+		TerraformDir:  "solutions/fscloud",
 		Region:        "us-south", // For FSCloud locking into us-south since that is where the dedicated host is provisioned
+		Prefix:        "fscloud",
 		ResourceGroup: resourceGroup,
-		TerraformVars: map[string]interface{}{
-			"access_tags": permanentResources["accessTags"],
-			// crn of the dedicated host
-			"environment_crn": permanentResources["dedicatedHostCrn"],
-		},
 	})
+
+	options.TerraformVars = map[string]interface{}{
+		"ibmcloud_api_key": options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"],
+		"access_tags":      permanentResources["accessTags"],
+		// crn of the dedicated host
+		"environment_crn":         permanentResources["dedicatedHostCrn"],
+		"existing_resource_group": true,
+		"resource_group_name":     options.ResourceGroup,
+		"instance_name":           options.Prefix,
```

Comment:
We probably want to pass a value for `database_config` here in order to test database creation